components/data-set-controls.vue

```diff
@@ -43,7 +43,7 @@
             id="layer-options-dropdown"
             v-model="selectedLayer"
             :options="items(dataset.layerOptions)"
-            :label="`Select ${dataset.name}`"
```

Comment:
Why these changes? (if you removed the interpolation, ` aren't required and you could use ", but that's a nitpick)

Reply:
When using Select ${dataset.name}, this leads to, for example:

![image](https://user-images.githubusercontent.com/50665134/106144708-ddc97480-6174-11eb-9f85-97ea6a235aea.png)

So underneath the dataset name "Maximum water level", it says "Select maximum water level". Which is not very useful. A text like "Select return period" would be more applicable here.

For the Tidal indicators, it says "Select Tidal indicators", which would be OK.

For the new CRUCIAL data, something like "Select analysis period" would be more applicable.

To change this into something more general, I have now changes this to "Select layer", which is less specific, but applicable to all of the above examples.